docs/integrations/spark.md

```diff
@@ -419,6 +419,29 @@ spark.hadoop.fs.s3a.bucket.example-repo.path.style.access true
 ```sql
 SELECT * FROM delta.`s3a://example-repo/main/datasets/delta-table/` LIMIT 100
 ```
+### ⚠️ Experimental: Pre-signed mode for S3A
+
+As of Hadoop 3.3.1 (and most likely, older versions as well), it is possible to use pre-signed URLs as return values from the lakeFS S3 Gateway.
+
+This has the immediate benefit of reducing the amount of traffic that has to go through the lakeFS server. To read more about pre-signed URLs, see [this guide](../reference/security/presigned-url.html).
+
+Here's an example Spark configuration to enable this support:
+
+```
+spark.hadoop.fs.s3a.impl shaded.databricks.org.apache.hadoop.fs.s3a.S3AFileSystem
+spark.hadoop.fs.s3a.bucket.example-repo.access.key AKIAIOSFODNN7EXAMPLE // The access key to your lakeFS server
+spark.hadoop.fs.s3a.bucket.example-repo.secret.key wJalrXUtnFEMI/K7MDENG/bPxRfiCYEXAMPLEKEY // The secret key to your lakeFS server
+spark.hadoop.fs.s3a.path.style.access true
+spark.hadoop.fs.s3a.bucket.example-repo.signing-algorithm QueryStringSignerType
+spark.hadoop.fs.s3a.bucket.example-repo.user.agent.prefix s3RedirectionSupport
```

Comment:
Can we also document that the the prefix only has to contain the word "s3RedirectionSupport", but that it doesn't have to come first?  I'm thinking about users who may _already_ tag their requests to identify their Spark cluster.